lib/cli/interactive-setup/deploy.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const Serverless = require('../../serverless');
-const { writeText, style, log } = require('@serverless/utils/log');
```

Comment:
I can revert the changes in this file if we would like but I feel like the changes here might be worth making 🤷

Reply:
@Danwakeem they're good, but best if either they're proposed in different PR (as they're not necessary related), or we can keep it in one PR, assuming we stick to semantic commits and PR can be rebased instead of squashing

Reply:
Okay yeah this is a good call. I Can open a separate PR 👍